.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,59 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ develop, master ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ develop ]
+  schedule:
+    - cron: '40 11 * * 5'
```

Comment:
why do we have to run it on schedule if we run it on push?

Reply:
New vulnerabilities could be added and this would catch them (that's my understanding)